index.js

```diff
@@ -256,7 +256,7 @@ export function requestSharedWebCredentials(): Promise<SharedWebCredentials> {
 export function setSharedWebCredentials(
   server: string,
   username: string,
-  password: string
+  password: ?string
```

Comment:
A `NULL` `password` passed to [`SecAddSharedWebCredential`](https://github.com/oblador/react-native-keychain/blob/master/RNKeychainManager/RNKeychainManager.m#L486) is used to [remove a shared password](https://developer.apple.com/documentation/security/1617986-secaddsharedwebcredential?language=objc)